.changeset/poor-kiwis-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': patch
+---
+
+Add `AlphaStack` component
```

Comment:
Instead of creating a new changeset, you can add to the existing one from the `layout-foundations-prototype` branch

It's `.changeset/red-cycles-allow.md`.